src/persist/src/indexed/cache.rs

```diff
@@ -137,8 +151,10 @@ impl<B: Blob> BlobCache<B> {
         let key = key.to_owned();
         // TODO: IO thread pool for persist instead of spawning one here.
```

Comment:
I tried to do this TODO while I was in here but hit some issues I didn't feel like working through quite yet /shrug